### PR TITLE
conic emissions

### PR DESCRIPTION
**Pull request by Define101**
conic emissions

### CODE REVIEW

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/conic.ts" should be below: 
 
 https://iili.io/Hwe6nVf.md.png

File: protocols/conic.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const start = 1649246400;
+const qty = 10_000_000;
+
+
+
+const conic: Protocol = {
+    vlCVX_Holders: manualCliff(start, qty * 0.1),
+    Community_raise: manualCliff(start, qty * 0.3),
+    Liquidity_Providers: manualLinear(start, start + periodToSeconds.month * 60, qty * 0.44),
```
Comment (waynebruce0x) [inline protocols/conic.ts:13]: Staking Omnipool LP tokens (25%) - 'emissions decrease by 60% per year'. So here we need to make a function, where we connect together manualLinear() sections (1 per year)

Rebalancing Curve pools (19%) - 'CNC received will be based on the amount deposited' - this is very hard to measure, so I'd exclude this part and write a note to say why

File: protocols/conic.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    Treasury: manualLinear(start, start + periodToSeconds.month * 12, qty * 0.05),
+    AMM_Stakers: manualLinear(start, start + periodToSeconds.week * 60, qty * 0.1),
```
Comment (waynebruce0x) [inline protocols/conic.ts:15]: AMM Stakers (10%) - 'inflation rate for these CNC rewards decreases by 60% every April. The inflation for AMM staking rewards originally started on April 6th, 2022'. This means we need a similar shape to 'Staking Omnipool LP tokens'. Could probably use the same function

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/conic.ts" should be below: 
 
 https://iili.io/HOCMLNI.md.png

**Comment by Define101** [comment]
Now seems good

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/arbitrum.ts" should be below: 
 
 https://iili.io/HOxS9yP.md.png

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/conic.ts" should be below: 
 
 https://iili.io/HOxSJu1.md.png

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/arbitrum.ts" should be below: 
 
 https://iili.io/HOxSXPj.md.png

**Comment by llamatester** [comment]
The emissions chart produced by "protocols/conic.ts" should be below: 
 
 https://iili.io/HOxSvN1.md.png

**Pull request merged**